.vscode/settings.json

```diff
@@ -12,7 +12,23 @@
   "files.associations": {
     "*.rbi": "ruby",
     "*.rbupdated": "ruby",
-    "*.rbedited": "ruby"
+    "*.rbedited": "ruby",
+    "vector": "cpp",
+    "*.tcc": "cpp",
+    "istream": "cpp",
+    "mutex": "cpp",
+    "ostream": "cpp",
+    "stop_token": "cpp",
+    "typeinfo": "cpp",
+    "array": "cpp",
+    "locale": "cpp",
+    "memory": "cpp",
+    "thread": "cpp",
+    "tuple": "cpp",
+    "variant": "cpp",
+    "functional": "cpp",
+    "utility": "cpp",
+    "stdexcept": "cpp"
```

Comment:
I don't want to have to have every header file defined in the standard library listed here.

Does something like `"**/include/c++/**"` work?

If not, can you move these to your user settings?

I'm not sure what causes you to need these settings. This is the list of extensions I have installed, and VS Code highlights C++ stdlib header files perfectly fine for me.

<img width="303" alt="Screen Shot 2022-09-07 at 12 29 25 PM" src="https://user-images.githubusercontent.com/5544532/188961308-a2a39b1c-1ef0-414a-bae7-43378edd2ea5.png">

Reply:
Oh wait whoops did not intend to commit this file, my mistake